code/modules/mob/mob_grab.dm

```diff
@@ -153,5 +153,5 @@
 				pulled.forceMove(xeno)
 				return TRUE
 		if(!(pulled in xeno.stomach_contents))
-			to_chat(xeno, SPAN_WARNING("You stop devouring \the [pulled]. \He probably tasted gross anyways."))
```

Comment:
Not sure \He works here without an object, and given the other alien messages using it "it" might just be more relevant